basics/a59.go

```diff
@@ -0,0 +1,35 @@
+// introduction to Arrays
+// This programs looks in to various array declarations
+package main
+
+import "fmt"
+
+func main() {
+	var a [3]int // initialized with array of 3 integers (0)
+	fmt.Println(a[0]) // will print 0
+	// print indices and elements
+	for i, v := range a {
+		fmt.Printf("index=%d, value=%d\n", i, v)
+	}
+	fmt.Println("Strings")
+	var b [3]string // an array of 3 sting characters
+	for i, v := range b {
+		fmt.Printf("index=%d, value=%s\n", i, v)
+	}
+	b[0] = "a"
+	b[1] = "b"
+	b[2] = "c"
+	for i, v := range b {
+		fmt.Printf("index=%d, value=%s\n", i, v)
+	}
```

Comment:
It would be easy to read the output if you could do a fmt.Println() between each section.